config/serialization.go

```diff
@@ -169,7 +176,7 @@ func Save(config map[string]map[string]interface{}) error {
 	}
 
 	// Write config file
-	configFile := configFile()
+	_ ,configFile := configFile()
```

Comment:
Ah okay. Then the places where I was using the string return would use io.writer instead... So I could pare down my configFile return to only io.ReadWriterCloser instead of both io.Reader and string, is that right?